2021-09-04 19:29:03 [scrapy] DEBUG: Crawled (200) <GET https://www.reliancedigital.in/bpl-109-22-cm-43-inch-full-hd-android-smart-tv-with-dolby-surround-sound-technology-43f-a4300/p/491893307> (referer: None)
2021-09-04 19:29:41 [scrapy] ERROR: Spider error processing <GET https://www.reliancedigital.in/bpl-109-22-cm-43-inch-full-hd-android-smart-tv-with-dolby-surround-sound-technology-43f-a4300/p/491893307> (referer: None)
Traceback (most recent call last):
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\utils\defer.py", line 120, in iter_errback
    yield next(it)
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\core\spidermw.py", line 56, in _evaluate_iterable
    for r in iterable:
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\core\spidermw.py", line 56, in _evaluate_iterable
    for r in iterable:
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 342, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\core\spidermw.py", line 56, in _evaluate_iterable
    for r in iterable:
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 40, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\core\spidermw.py", line 56, in _evaluate_iterable
    for r in iterable:
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\scrapy\core\spidermw.py", line 56, in _evaluate_iterable
    for r in iterable:
  File "C:\Users\G RAJA\Desktop\scrapy_mongo\scraper\scraper\ajar\ajar\spiders\relianceimages.py", line 51, in parse
    touchactions.tap(clicer).perform()
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\selenium\webdriver\common\touch_actions.py", line 47, in perform
    action()
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\selenium\webdriver\common\touch_actions.py", line 56, in <lambda>
    self._actions.append(lambda: self._driver.execute(
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "c:\users\graja~1\desktop\scrapy~2\scraper\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown command: Cannot call non W3C standard command while in W3C mode

2021-09-04 19:29:41 [scrapy] INFO: Closing spider (finished)
2021-09-04 19:29:41 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 384,
 'downloader/request_count': 1,
 'downloader/request_method_count/GET': 1,
 'downloader/response_bytes': 381495,
 'downloader/response_count': 1,
 'downloader/response_status_count/200': 1,
 'elapsed_time_seconds': 48.644528,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2021, 9, 4, 13, 59, 41, 801518),
 'log_count/DEBUG': 20,
 'log_count/ERROR': 1,
 'log_count/INFO': 8,
 'response_received_count': 1,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'spider_exceptions/WebDriverException': 1,
 'start_time': datetime.datetime(2021, 9, 4, 13, 58, 53, 156990)}
2021-09-04 19:29:41 [scrapy] INFO: Spider closed (finished)
